src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="EnsureNETCoreAppRuntime" 
+          Condition="'$(RuntimeIdentifier)' != '' AND '$(EnsureNETCoreAppRuntime)' != 'false'" 
+          AfterTargets="RunResolvePackageDependencies">
+    <ItemGroup>
+      <_netCoreAppPackageDependencies 
+          Include="@(PackageDependencies->WithMetadataValue('ParentPackage', '$(MSBuildThisFileName)/$(_Microsoft_NETCore_App_Version)'))" />
+      <_activeRIDNetCoreAppPackageDependencies 
```

Comment:
I've created my own items and avoided using all items/properties that do similar things in Microsoft.PackageDependencyResolution.targets since those are all private ('_').